src/common/file_utils.cpp

```diff
@@ -118,12 +119,9 @@ void FileUtils::removeFileIfExists(const std::string& path) {
 
 std::vector<std::string> FileUtils::globFilePath(const std::string& path) {
     std::vector<std::string> result;
-    glob_t globResult;
-    glob(path.c_str(), GLOB_TILDE, nullptr, &globResult);
-    for (auto i = 0u; i < globResult.gl_pathc; ++i) {
-        result.emplace_back(globResult.gl_pathv[i]);
+    for (auto &resultPath: glob::glob(path)) {
```

Comment:
I think it is better to save the result of `glob::glob(path)` in a local variable, otherwise, we will call glob::glob(path) in each iteration.